src/main/java/roomescape/reservation/controller/request/ReservationCreateRequest.java

```diff
@@ -0,0 +1,12 @@
+package roomescape.reservation.controller.request;
+
+import jakarta.validation.constraints.NotNull;
+import java.time.LocalDate;
+
+public record ReservationCreateRequest(
+        @NotNull String name,
```

Comment:
1. 어노테이션으로 검증 진행 시 메세지를 꼭 입력해주세요 😄 

```suggestion
        @NotNull(message = "예약 날짜를 입력해주세요.")
        String name,
```

2. 해당 어노테이션 검증 실패 시 (null 입력 시) 발생하는 예외도 `@ControllerAdvice` 에서 처리해보면 어떨까요~?

Reply:
감사합니다! null이 입력됐을 때 발생하는 `MethodArgumentNotValidException`를 처리하는 메서드를RoomescapeExceptionHandler에 추가했어요!